torchci/pages/metrics.tsx

```diff
@@ -1195,19 +1205,24 @@ export default function Page() {
             title={"LF vs Meta: Duration increase ratio"}
             queryName={"lf_rollover_health"}
             queryCollection={"metrics"}
-            queryParams={[
-              {
-                name: "days_ago",
-                type: "int",
-                value: timeRange,
-              },
-            ]}
+            queryParams={
+              useClickHouse
+                ? { ...timeParamsClickHouse, days_ago: timeRange }
+                : [
+                    {
+                      name: "days_ago",
+                      type: "int",
+                      value: timeRange,
+                    },
+                  ]
+            }
             granularity={"day"}
             timeFieldName={"bucket"}
             yAxisLabel="increase ratio"
             yAxisFieldName={"success_duration_increase_ratio"}
             yAxisRenderer={(value) => value}
             groupByFieldName={"job_name"}
+            useClickHouse={useClickHouse}
```

Comment:
I just realized these 3 panels use the same values, is it worth it to query once initially and reuse the values?  Or maybe the caching is good enough to handle it

Reply:
There's some smarts that are already there under the hood, perhaps automated by React. If you look at the network requests only one request is actually made to the query, and it populates all three charts simultaneously